Sources/Models/SlackCommandResponse.swift

```diff
@@ -0,0 +1,22 @@
+// See: https://api.slack.com/slash-commands
+
+enum SlackCommandResponseType {
+    case Ephemeral // Visible only to user that issued the command
+    case InChannel // Visible to all members of the channel in which user typed the command
+}
+
+struct SlackCommandResponse {
+    
+    // If you want the user to see their slash command displayed in the channel and
+    // no response, generate a response with responseType = .InChannel and no text
+    
+    let responseType: SlackCommandResponseType
+    let text: String?
+    let attachments: [String: String]? // Used for further customisation of a message
```

Comment:
I'm a little bit uncertain about this one.
Attachments are rather complex (https://api.slack.com/docs/attachments) and they could and probably will be abstracted by something else in the future, so I would shield myself against that future change with typealias or protocol. On the other hand they are not needed to implement our current functionality and could be omitted altogether.

I'd love to hear what other team members think about it.